addons/hivedr/src/main/java/org/apache/falcon/hive/util/FileUtils.java

```diff
@@ -34,8 +34,10 @@
 public final class FileUtils {
 
     public static final String DEFAULT_EVENT_STORE_PATH = StringUtils.removeEnd(DRStatusStore
-            .BASE_DEFAULT_STORE_PATH,  File.separator) + File.separator + "Events";
+            .BASE_DEFAULT_STORE_PATH,  File.separator) + File.separator + "Events" + File.separator;
```

Comment:
@yzheng-hortonworks  That is a good point. Since it is a single line change, I will change it while commit to unblock 0.10